From 238893d8cd6f9c2616a05ab521a29651a17a38c2 Mon Sep 17 00:00:00 2001
From: Christian Heusel <christian@heusel.eu>
Date: Sat, 20 Apr 2024 11:07:32 +0200
Subject: [PATCH] fix build failure due to typo in the build script

The fixed commit had a typo where it would try to call a nonexistant
function instead of the .contains method of the relevant array.
This was only made visible when the "other" loaders were enabled so it
is triggered all the time.

This was made visible when working on
https://gitlab.archlinux.org/archlinux/packaging/packages/gdk-pixbuf2/-/merge_requests/1

Fixes: 81a77399a ("Update the testsuite")
Signed-off-by: Christian Heusel <christian@heusel.eu>
---
 tests/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index 78d0ad92e..78ae43af1 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -71,7 +71,7 @@ installed_tests = {
   'pixbuf-construction': { 'suites': ['conform'], },
   'animation': {
     'suites': ['format'],
-    'skip': not (enabled_loaders.contains('ani') and enabled_loaders_contains('gif')),
+    'skip': not (enabled_loaders.contains('ani') and enabled_loaders.contains('gif')),
   },
   'cve-2015-4491': {
     'suites': ['security'],
-- 
GitLab

